### ISSUE TITLE
Customize About box

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/Viewers#52]
I think we should do the following:

* OHIF Viewer -> IDC Viewer
* URL should probably point to the fork
* Visit the forum -> https://discourse.canceridc.dev/
* Report the issue -> https://discourse.canceridc.dev/

Can we add a line saying that IDC Viewer is based on / customized fork of the OHIF Viewer, and add a link to upstream?

![image](https://user-images.githubusercontent.com/313942/101805332-a299b600-3ae0-11eb-8259-1c79f398907c.png)

I want to give full credit to OHIF, but I also don't want to confuse the users, and I want the About box to be helpful in tracking down the actual code that is used in the viewer the user is seeing.

@pieper @swederik @Punzo what do you think?

**Comment by Punzo** [comment]
I think the changes make sense, and regarding the credit a line to give credit to OIHF is reasonable. Maybe in the credit line should be present the link of the OIHF master repo? what do you think @igoroctaviano, @swederik and @andrebot?

**Comment by igoroctaviano** [comment]
Makes sense, i would change Version Number => OHIF Version Number or maybe create a new section below the current one and name it OHIF Version Information?

**Comment by fedorov** [comment]
> Makes sense, i would change Version Number => OHIF Version Number or maybe create a new section below the current one and name it OHIF Version Information?

Yes, I think it is important to know both the upstream base version, and the specific fork version used.
